config/validator/validation.yaml

```diff
@@ -40,6 +40,12 @@ App\Form\Model\Registration:
         plainPassword:
             - App\Validator\Constraints\PasswordPolicy: ~
 
+App\Form\Model\RecoveryTokenAck:
```

Comment:
I would call this Form and Model `RegistrationRecoveryToken`

Reply:
It's used both by RegistrationController and by RecoveryController, so I decided to give it a more general name.

Reply:
But you named it now for the only property not for the purpose of the form.

Reply:
Mh, not sure whether I understand your point. The model is called `RecoveryTokenAck` because it's a model for the form to acknowledge (ack) the recovery token. What's wrong with the name in your opinion?